src/Extensions/AzureBlobStorage.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Remove this and add the AzureBlobStorage project to the dotnet-monitor.sln file if it already doesn't exist.